apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -244,6 +245,8 @@ export const ModerationNotification = (props) => {
   } = notification
   const markReadText = django.gettext('Mark as read')
   const markUnreadText = django.gettext('Mark as unread')
+  // FIXME after userReport added to serializer is can be added above and code should work
+  const userReport = ['Sollicitudin gravida purus et dolor vel rutrum varius adipiscing metus dolor quam gravida vivamus ac bibendum magna tincidunt mi nec elit phasellus nisl id erat.', 'This is another of those']
```

Comment:
@hklarner this is where you would add the user reports once they are in the serializer